demo/angular-grid.html

```diff
@@ -1,11 +1,11 @@
 <vaadin-grid #grid (selected-items-changed)="onSelect()">
   <table>
     <colgroup>
-      <col width=80 header-text="" name="user.picture.thumbnail">
-      <col header-text="Gender" name="user.gender">
-      <col header-text="First name" name="user.name.first">
-      <col header-text="Last name" name="user.name.last">
-      <col header-text="Email" name="user.email">
+      <col width=80 name="user.picture.thumbnail">
+      <col name="user.gender">
+      <col name="user.name.first">
+      <col name="user.name.last">
+      <col name="user.email">
```

Comment:
Although we agreed while planning the ticket that with demos that have names defined, we don't append a thead, it might be weird that that the picture column says "Thumbnail". Discussed this with @tehapo earlier also. Any comments @jounik ?